scripts/build.sh

```diff
@@ -12,7 +12,6 @@ yarn build:tsc
 echo "Fix path resolution"
 dist_folder="./dist"
 # Go through all ts and js files in the dist folder
-find "$dist_folder" -type f \( -name "*.ts" -o -name "*.js" \) -exec sed -i 's|require("@/|require("src/|g' {} +
-
+find "$dist_folder" -type f \( -name "*.ts" -o -name "*.js" \) -exec sed -i 's|require("@/|require("../../|g' {} +
```

Comment:
I have some doubts that all @/ refer to the ../.. forlder - there can be different nesting levels